src/xdocs/google_style.xml

```diff
@@ -2302,24 +2302,62 @@
         <p>
           It is possible to suppress some violations by embeded filters
           <a href="config_filters.html#SuppressionFilter">
-            SuppressionFilter</a>
-          and
+            SuppressionFilter</a>,
           <a href="config_filters.html#SuppressionXpathFilter">
-            SuppressionXpathFilter</a>.
+            SuppressionXpathFilter</a>,
+          <a href="config_filters.html#SuppressWithNearbyCommentFilter">
+            SuppressWithNearbyCommentFilter</a>,
+          <a href="config_filters.html#SuppressionCommentFilter">
+            SuppressionCommentFilter</a>
+          and
+          <a href="config_filters.html#SuppressWarningsFilter">
+            SuppressWarningsFilter</a>
+          in conjunction with
+          <a href="config_filters.html#SuppressWarningsHolder">
+            SuppressWarningsHolder</a>.
```

Comment:
Please remove SuppressWarningsHolder, it is very hacky module that we are not very proud of. It is helper module, it is not a filter.